src/Testing/TestCase.php

```diff
@@ -8,16 +8,19 @@
 use Tempest\AppConfig;
 use Tempest\Application\Kernel;
 use Tempest\Container\Container;
-use Tempest\Testing\Http\TestsHttpRequests;
+use Tempest\Testing\Console\ConsoleCommandTester;
+use Tempest\Testing\Http\HttpRouterTester;
 
 abstract class TestCase extends BaseTestCase
```

Comment:
Do we plan on migrating all our tests to extend from this TestCase?

Reply:
I think we need to split out our tests into `Unit` and `Integration` tests. Probably all `Integration` tests would extend `Tempest\Testing\TestCase`.

What are your thoughts on that?

Reply:
I'm fine with that, but I'd make it a rule that as soon as the framework boots, it's an integration test. It means that 99% of tests are integration tests right now.